src/nyc_trees/apps/home/urls.py

```diff
@@ -17,7 +17,9 @@
     url(r'^jobs/(?P<job_id>\d+)/$', r.retrieve_job_status,
         name='retrieve_job_status'),
 
-    url(r'^training/$', r.training_list_page),
+    url(r'^training/$', r.training_list_page, name="training_list_page"),
+
+    url(r'^training/groups_to_follow/$', r.groups_to_follow),
```

Comment:
This may be a flatpages thing but you get a 404 if the trailing slash is omitted. Don't know what we can do about that.